### PR TITLE
Add a missing author to the list

### PR DESCRIPTION
**Pull request by JonathanGregory**
See issue #483 for discussion of these changes.

# Release checklist
- [NA] Authors updated in `cf-conventions.adoc`?
- [NA] Next version in `cf-conventions.adoc` up to date? Versioning inspired by [SemVer](https://semver.org).
- [Y] `history.adoc` up to date?
- [NA] Conformance document up-to-date?

### CODE REVIEW

File: cf-conventions.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+Brian{nbsp}Eaton; Jonathan{nbsp}Gregory; Bob{nbsp}Drach; Karl{nbsp}Taylor; Steve{nbsp}Hankin; Jon{nbsp}Blower; John{nbsp}Caron; Rich{nbsp}Signell; Phil{nbsp}Bentley; Greg{nbsp}Rappa; Heinke{nbsp}Höck; Alison{nbsp}Pamment; Martin{nbsp}Juckes; Martin{nbsp}Raspaud; Randy{nbsp}Horne; Timothy{nbsp}Whiteaker; David{nbsp}Blodgett; Charlie{nbsp}Zender; Daniel{nbsp}Lee; David{nbsp}Hassell; Alan{nbsp}D.{nbsp}Snow; Tobias{nbsp}Kölling; Dave{nbsp}Allured; Aleksandar{nbsp}Jelenak; Anders{nbsp}Meier{nbsp}Soerensen; Lucile{nbsp}Gaultier; Sylvain{nbsp}Herlédan; Fernando{nbsp}Manzano; Lars{nbsp}Bärring; Chris{nbsp}Barker
```
Comment (ChrisBarker-NOAA) [inline cf-conventions.adoc:3]: Could you make that "Christopher Barker" -- I usually use my full name in professional contexts.

Thanks,
-CHB

**Comment by davidhassell** [comment]
Hi - Chris was added in the merged [UGRID PR](https://github.com/cf-convention/cf-conventions/pull/459/files#diff-cce1a5a512afe46cd98d51268562ded8c5f887d6a4e1882c5ad8b3725a86a99e) (albeit as Chris, not Christopher - sorry), so I was just wondering how comes the original addition seems to have disappeared?

**Comment by JonathanGregory** [comment]
> Hi - Chris was added in the merged [UGRID PR](https://github.com/cf-convention/cf-conventions/pull/459/files#diff-cce1a5a512afe46cd98d51268562ded8c5f887d6a4e1882c5ad8b3725a86a99e) (albeit as Chris, not Christopher - sorry), so I was just wondering how comes the original addition seems to have disappeared?

Yes, it's surprising, but easily fixed!

File: cf-conventions.adoc (hunk 1 above)
Comment (erget) [reply cf-conventions.adoc]: Corrected in 5a898fdebee63fe0361b125c9146b64dc0d3b12f

**Pull request merged**